### DISCUSSION TITLE
I think the conditions for relational collections don't work

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#13742] [Archive]
hey i'm trying to set conditions for a field that is referenced in a relational collection. Unfortunately, it doesn't work for me. Am I setting something wrong or is it an application error?

https://user-images.githubusercontent.com/90528895/172067376-c1114317-85d5-4867-ab2d-eba11dc4fdae.mp4

any suggestion?

**Comment by azrikahar** [comment]
This is a known issue and it's being tracked #8228. This comment elaborates more on what's happening here: https://github.com/directus/directus/issues/7423#issuecomment-902293310

Additional ref: #11844, #7123

**Reply by koksikus** [reply]
Thank you
